docs/modeling/example-fitting-constraints.rst

```diff
@@ -47,38 +63,91 @@ Fitters support constrained fitting.
       ...                          n_models=2, fixed={'c0': True})
 
 
-- A parameter can be `~astropy.modeling.Parameter.tied` (linked to
-  another parameter). This can be done in two ways::
-
-      >>> def tiedfunc(g1):
-      ...    mean = 3 * g1.stddev
-      ...    return mean
-      >>> g1 = models.Gaussian1D(amplitude=10., mean=3, stddev=.5,
-      ...                        tied={'mean': tiedfunc})
-
-  or::
-
-      >>> g1 = models.Gaussian1D(amplitude=10., mean=3, stddev=.5)
-      >>> g1.mean.tied = tiedfunc
+Bounded Constraints
+-------------------
 
 Bounded fitting is supported through the ``bounds`` arguments to models or by
 setting `~astropy.modeling.Parameter.min` and `~astropy.modeling.Parameter.max`
 attributes on a parameter.  Bounds for the
 `~astropy.modeling.fitting.LevMarLSQFitter` are always exactly satisfied--if
 the value of the parameter is outside the fitting interval, it will be reset to
-the value at the bounds. The `~astropy.modeling.fitting.SLSQPLSQFitter` handles
-bounds internally.
+the value at the bounds. The `~astropy.modeling.fitting.SLSQPLSQFitter` optimiztion
+algorithm handles bounds internally.
 
-- Different fitters support different types of constraints::
+.. _tied:
 
-    >>> fitting.LinearLSQFitter.supported_constraints
-    ['fixed']
-    >>> fitting.LevMarLSQFitter.supported_constraints
-    ['fixed', 'tied', 'bounds']
-    >>> fitting.SLSQPLSQFitter.supported_constraints
-    ['bounds', 'eqcons', 'ineqcons', 'fixed', 'tied']
+Tied Constraints
+----------------
+
+The `~astropy.modeling.Parameter.tied` constraint is often useful with :ref:`Compound models <compound-models-intro>`.
+In this example we will read a spectrum from a file called ``spec.txt``
+and fit Gaussians to the lines simultaneously while linking the flux of the OIII_1 and OIII_2 lines.
+
+.. plot::
+    :include-source:
+       
+    >>> import numpy as np
+    >>> from astropy.io import ascii
+    >>> from astropy.utils.data import get_pkg_data_filename
+    >>> from astropy.modeling import models, fitting
+    >>> fname = get_pkg_data_filename('data/spec.txt', package='astropy.modeling.tests')
+    >>> spec = ascii.read(fname)
+    >>> wave = spec['lambda']
+    >>> flux = spec['flux']
+    
+    Use the rest wavelengths of known lines as initial values for the fit.
+
+
+    >>> Hbeta = 4862.721
```

Comment:
Would be nice to use Quantity throughout to show people how modeling/fitting framework can integrate with Quantity for a realistic astronomical use case like this.

Reply:
Sure. But it will have to wait for the future - when Quantities work better with fitting.